pkg/templateservicebroker/brokerapi/templateservicebroker/catalog.go

```diff
@@ -0,0 +1,48 @@
+package templateservicebroker
+
+import (
+	"net/http"
+	"strings"
+
+	templateapi "github.com/openshift/origin/pkg/template/api"
+	"github.com/openshift/origin/pkg/templateservicebroker/brokerapi"
+	"k8s.io/kubernetes/pkg/api"
+)
+
+func serviceFromTemplate(template *templateapi.Template) *brokerapi.Service {
+	metadata := make(map[string]interface{})
+	if displayName, ok := template.Annotations["openshift.io/display-name"]; ok {
+		metadata["displayName"] = displayName
+	}
+	// TODO: metadata[imageUrl"]
+	// TODO: metadata[longDescription"]
+	// TODO: metadata[providerDisplayName"]
+	// TODO: metadata[documentationUrl"]
+	// TODO: metadata[supportUrl"]
+	// TODO: list template parameters (https://github.com/openservicebrokerapi/servicebroker/issues/59)
```

Comment:
see the other issue/PR i have belatedly tagged you into for the names of these other annotations.